dpnp/backend.pxd

```diff
@@ -113,15 +113,32 @@ cdef extern from "backend/backend_iface.hpp" namespace "QueueOptions":  # need t
         CPU_SELECTOR
         GPU_SELECTOR
 
+cdef extern from "backend/backend_iface.hpp":
+    cdef enum EngineOptions:
+            ARS5 = 0
+            MCG32M1 = 1
```

Comment:
using " = 1" here meaningless because it lost connection between C++ enum and Enum defined here.
It must get the same values as in C++